includes/api/class-wc-rest-product-reviews-controller.php

```diff
@@ -143,6 +241,190 @@ public function get_item( $request ) {
 		return $response;
 	}
 
+
+	/**
+	 * Create a product review.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_Error|WP_REST_Response
+	 */
+	public function create_item( $request ) {
+		$product = get_post( (int) $request['product_id'] );
+
+		if ( empty( $product->post_type ) || 'product' !== $product->post_type ) {
+			return new WP_Error( 'woocommerce_rest_product_invalid_id', __( 'Invalid product ID.', 'woocommerce' ), array( 'status' => 404 ) );
+		}
+
+		if ( empty( $request['review'] ) ) {
+			return new WP_Error( 'woocommerce_rest_product_review_invalid_review', __( 'Product review content is required.', 'woocommerce' ), array( 'status' => 400 ) );
+		}
+
+		if ( empty( $request['name'] ) ) {
+			return new WP_Error( 'woocommerce_rest_product_review_invalid_name', __( 'Product review author name is required.', 'woocommerce' ), array( 'status' => 400 ) );
+		}
+
+		if ( empty( $request['email'] ) ) {
+			return new WP_Error( 'woocommerce_rest_product_review_invalid_email', __( 'Product review author email is required.', 'woocommerce' ), array( 'status' => 400 ) );
+		}
+
+		$data = array(
+			'comment_post_ID'      => $product->ID,
+			'comment_author'       => $request['name'],
+			'comment_author_email' => $request['email'],
+			'comment_content'      => $request['review'],
+			'comment_approved'     => 1,
+			'comment_type'         => 'review',
+		);
+		$product_review_id = wp_insert_comment( $data );
```

Comment:
This can return false.
We can do like WP-API.

```php
		$comment_id = wp_insert_comment( $prepared_comment );
		if ( ! $comment_id ) {
			return new WP_Error( 'rest_comment_failed_create', __( 'Creating comment failed.' ), array( 'status' => 500 ) );
		}
```